packages/block-library/src/image/edit.native.js

```diff
@@ -443,13 +502,13 @@ export class ImageEdit extends Component {
 		} = this.props;
 		const { align, url, alt, id, sizeSlug, className } = attributes;
 
+		const isFeaturedImage = featuredImageId === attributes.id;
```

Comment:
👋 Siobhan! I see the [same line in trunk already](https://github.com/WordPress/gutenberg/blob/f14962f283f63039be1e790f552489a1705d2928/packages/block-library/src/image/edit.native.js#L451) so, I assume the PR needs some updating from trunk? While at it, since by now you have write access to the Gutenberg repo, it makes sense to re-open this PR as one directly on the repo instead of the fork, that will speed up the review process, wdyt?

Reply:
Thank you for pointing that out, it looks like the line got moved about at some point in this PR, I'll rework that.

> it makes sense to re-open this PR as one directly on the repo instead of the fork, that will speed up the review process, wdyt?

This makes sense to me and will likely simplify the process of cleaning up the commit history! I'll get about doing that and ping you when the new PR is done. :)